src/main/java/duke/DateTimeParser.java

```diff
@@ -0,0 +1,67 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class is used to parse the date and time from the user input.
+ */
+public class DateTimeParser {
+    // Note that we take "MMM-dd-yyyy HHmm a" and "MMM-d-yyyy HHmm a" as valid date and time formats because
+    // we store the date and time in this format in the storage file. Therefore, in order to convert it back
+    // to DateTime, we need to take these formats as valid.
+    public static final String[] VALID_DATE_TIME_FORMAT = {"dd/MM/yyyy HHmm", "d/MM/yyyy HHmm",
+        "MMM-dd-yyyy HHmm a", "MMM-d-yyyy HHmm a"};
+
+    public static final String[] VALID_DATE_TIME_FORMAT_OUTPUT = {"MMM-dd-yyyy HHmm a", "MMM-d-yyyy HHmm a"};
+
+    /**
+     * Checks if the input is a valid date and time format
```

Comment:
```suggestion
     * Checks if the input is a valid date and time format.
```
Should you place a full stop at the end here?
I noticed the same issue in other places too.